examples/tasks/mock-statistics-file.json

```diff
@@ -0,0 +1,40 @@
+{
+	"version": 1,
+	"schedule": {
+		"type": "simple",
+		"interval": "1s"
+	},
+	"max-failures": 10,
+	"workflow": {
+		"collect": {
+			"metrics": {
+				"/intel/mock/foo": {},
```

Comment:
these are mock collector metrics. It seems that the shell scripts loaded psutil. Do you like to use some of psutil metrics?